repository_service_tuf/cli/admin/ceremony.py

```diff
@@ -457,7 +457,7 @@ def _check_server(settings):
     return headers
 
 
-def _bootstrap(server, headers, json_payload):
+def _bootstrap(server, headers, json_payload) -> Optional[str]:
```

Comment:
I think we can do the same simplification as in `_check_server` and remove the `else` statement on line 481.